less/button.less

```diff
@@ -155,3 +155,37 @@ span.menuButton {
     font-size: 14px;
   }
 }
+
+// for about:preferences
```

Comment:
The buttons under this section should be removed if necessary. especially from L171 to L176.